validator/src/main.rs

```diff
@@ -562,13 +575,6 @@ pub fn main() {
         }
     };
 
-    solana_logger::setup_with_filter(
```

Comment:
;) No wonder I couldn't see `debug!`s, `warn!`s, etc in the preceding large chunk of arg handling code....

Reply:
Yeah, I recently moved it here from where you moved it back to because I wanted the arg parsing code to use println/eprintln only.  The log crate then only gets used after the `--log` argument is processed and the log destination is configured.

Reply:
The entire argument parsing in this function eventually needs to be split out into it's own function, so the clippy complexity suppression can be removed

Reply:
Thanks for detailed explanation. I'll then just resotre this change.